### PR TITLE
golang version of `mwctl` w/ corresponding build

### PR DESCRIPTION
**Pull request by eevans**
The Makefile here is cribbed from blubber
(https://phabricator.wikimedia.org/source/blubber/browse/master/Makefile),
adapted to cross-compile binaries for linux, windows and darwin.  It uses a
locally overridden `GOPATH` (mwctl/.build) and so when you build, it has to
fetch and build anything that isn't already in `GOROOT`, which includes the
relevant parts of standard lib for non-native platforms.

TL;DR The non-native builds will take a bit longer to build.  You should use
the apropos make target (i.e. `make linux`) while iterating development of
mwctl, and then build all to check-in any changes.

### CODE REVIEW

File: mwctl/Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+export GOPATH=$(CURDIR)/.build
+
+GO ?= /usr/bin/go
```
Comment (Pchelolo) [inline mwctl/Makefile:4]: Not really... On OS X it's `/usr/local/bin/go`, so perhaps just do `which go`?

Comment (eevans) [reply mwctl/Makefile:4]: Yeah, I thought about this, which is why it is `?=`, so that you can do `GO=/usr/local/bin/go make darwin`.

I'm of two minds here: 

The Neck Beard in me says that this is a compiler, that it's not uncommon for people to have multiple versions installed, and the one in `PATH` might not be the Right One.  Basically, that explicit >> implicit.

The pragmatist in me however realizes that I might have to defend this principal to everyone who (understandably) is irritated by having to type an extra `GO=...` on each invocation.

So perhaps you are right and we should use `which`. :)

**Comment by d00rman** [comment]
Closing in favour of wikimedia/mwctl#1
